test/extended/cli/explain.go

```diff
@@ -586,6 +592,12 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 				e2e.Logf("Checking %s, Field=%s...", st.gv, st.field)
 				resource := strings.Split(st.field, ".")
 				gvr := st.gv.WithResource(resource[0])
+				exist, err := exutil.DoesApiResourceExist(oc.AdminConfig(), resource[0], groupName)
+				o.Expect(err).NotTo(o.HaveOccurred())
+				if !exist {
+					e2e.Logf("Resource %s does not exist, skipping", gvr)
```

Comment:
This will show the test as passed, not skipped. This needs to be `g.Skip` instead.

Reply:
Can't skip the whole test, as this is only skipping specific resources from an API group. Using `g.Skip` would mean skipping the entire API group.